conan/tools/cmake/toolchain/blocks.py

```diff
@@ -5,7 +5,7 @@
 
 from jinja2 import Template
 
-from conan.tools.apple.apple import get_apple_sdk_fullname
+from conan.tools.apple.apple import get_apple_sdk_fullname, _to_apple_archs, _is_universal_arch
```

Comment:
Maybe try to avoid the private imports? Maybe moving them to internal, or making them public (within the private sub-folder)